### ISSUE TITLE
AccountRow description should indicate which limits are active

### ISSUE THREAD

**Issue by danrabbit** [elementary/switchboard-plug-parental-controls#114]
The username is pretty useless here. Instead we should indicate with limits are active

**Comment by cassidyjames** [comment]
I think we do need a way to distinguish accounts though, maybe on tooltip? It would be stupid, but you can absolutely have two accounts with the exact same avatar and "real name."

**Comment by danrabbit** [comment]
@cassidyjames Yeah a tooltip makes sense here to me. Can you file that as a separate issue?
